op-e2e/actions/proofs/sequence_window_expiry_test.go

```diff
@@ -5,26 +5,47 @@ import (
 
 	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
 	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils"
 	"github.com/ethereum-optimism/optimism/op-program/client/claim"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/require"
 )
 
-// Run a test that proves a deposit-only block generated due to sequence window expiry.
+// Run a test that proves a deposit-only block generated due to sequence window expiry,
+// and then recovers the chain using sequencer recover mode.
```

Comment:
Do you think we could make this a 2nd test? If we're looking at test output, and `SequenceWindowExpireTest` is failing, I would like to know if it's the deposit-only that's failing, or the recovery mode.